files/build_templates/multi_instance/swss@.service.j2

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=switch state service
+Requires=database@%i.service 
+#updategraph.service
```

Comment:
Worried about code redundancy between this and swss.service.j2. Can you unify them, for example, single instance just call this one with an empty string parameter.

Reply:
Fixed, created single template file.